website/docs/r/kms_key_with_policy_overrides.html.markdown

```diff
@@ -0,0 +1,145 @@
+---
+
+subcategory: "Key Management Service"
+layout: "ibm"
+page_title: "IBM : kms-key-with-policy-overrides"
+description: |-
+  Manages IBM hs-crypto and KMS key with specified key material and key policies. This API overrides the policy configurations set at instance level with policies provided in the payload.
```

Comment:
This is not an API. Also, we can say overrides the policy  configuration at the time of key creation.